app/assets/javascripts/delete_idea.js

```diff
@@ -19,6 +20,9 @@ function deleteRow(urlString, removeRow) {
     url:      urlString,
     dataType: 'json',
     success:  function() {
+      // This sort of nesting can get ugly pretty fast, especially if the 
+      // html changes - you won't always be in control of it. Try and think 
+      // of another way to do this.
```

Comment:
This is one of the things that gets me the most. The event.target is the delete button, so I don't see how you would get there other than that. I could maybe use jQuery but i'm sure this is one of those times when 'this' isn't what i want it to be. Can't believe I didn't realize we could have a conversation like this.